runelite-client/src/main/java/net/runelite/client/plugins/worldmap/FarmingPatchLocation.java

```diff
@@ -0,0 +1,57 @@
+package net.runelite.client.plugins.worldmap;
+
+import lombok.Getter;
+import net.runelite.api.coords.WorldPoint;
+
+@Getter
+enum FarmingPatchLocation
+{
+	AL_KHARID_CACTUS("Cactus", new WorldPoint(3313, 3201, 0)),
```

Comment:
Is there a reason every single patch has its own enum instead of an enum containing all trees, hops, spirit trees, and so on?

Reply:
I Can refactor like this? Or just keep it like it is?
```
	SEAWEED("Seaweed", Arrays.asList(
		new WorldPoint(3730, 10271, 0)
	)),
	SPIRIT_TREE("Spirit Tree", Arrays.asList(
		new WorldPoint(3056, 3259, 0),
		new WorldPoint(1690, 3540, 0),
		new WorldPoint(3614, 3856, 0),
		new WorldPoint(2799, 3205, 0)
	)),
	TREE("Tree", Arrays.asList(
		new WorldPoint(3226, 3457, 0),
		new WorldPoint(2933, 3436, 0),
		new WorldPoint(3189, 3233, 0),
		new WorldPoint(2434, 3418, 0),
		new WorldPoint(3005, 3375, 0)
	));
```

Reply:
I think changing it to
`FarmingPatchLocation(String description, WorldPoint... locations)`
and
`SPIRIT_TREE("Spirit Tree",	new WorldPoint(3056, 3259, 0),
		new WorldPoint(1690, 3540, 0),
		new WorldPoint(3614, 3856, 0),
		new WorldPoint(2799, 3205, 0))`
would be the best solution